src/kili/services/plugins/upload.py

```diff
@@ -118,14 +138,65 @@ def _create_plugin_runner(self):
         result = self.auth.client.execute(GQL_CREATE_PLUGIN_RUNNER, variables)
         return format_result("data", result)
 
+    def _check_plugin_runner_status(self, update=False):
+        """
+        Check the status of a plugin's runner until it is active
+        """
+
+        logger = self.get_logger()
+
+        action = "updated" if update else "created"
+
+        logger.info(f"Plugin is being {action}... This should take approximately 3 minutes.")
+
+        n_tries = 0
+        status = None
+
+        logger = self.get_logger("INFO")
+
+        while n_tries < NUMBER_TRIES_RUNNER_STATUS:
+            status = self._get_plugin_runner_status()
+
+            if status == "ACTIVE":
+                break
+
+            logger.info(f"Status : {status}...")
+
+            n_tries += 1
+            time.sleep(15)
+
+        if status == "DEPLOYING" and n_tries == 20:
+            raise Exception("Limit of number of tries exceeded.")
```

Comment:
I think our warning should give a hint for the user on what to do.
Perhaps something like : 
We could not check your plugin was deployed in time. Please check your plugin's code and try again ? 
Or expose them the method `kili.get_plugin_status` and hint them to wait and use it ?

Reply:
Good idea, I'll expose `kili.get_plugin_status` and add the hint

Reply:
@p-desaintchamas  Updated the message and exposed `kili.get_plugin_status`. Created a class for queries related to plugins to add the new method and also moved `get_logs` since it was in `mutations` before